tests/integration/annotation_import/test_model_run.py

```diff
@@ -56,6 +56,24 @@ def test_model_run_delete(client, model_run):
     assert len(before) == len(after) + 1
 
 
+def test_model_run_update_config(model_run_with_training_metadata):
+    new_config = {"batch_size": 2000}
+    res = model_run_with_training_metadata.update_config(new_config)
+    assert res["trainingMetadata"]["batchSize"] == new_config["batch_size"]
+
+
+def test_model_run_reset_config(model_run_with_training_metadata):
+    res = model_run_with_training_metadata.reset_config()
+    assert res["trainingMetadata"] == None
```

Comment:
nit: When you check if an object value is None you should use the identify check with the `is` operator. Technically the equality operator of whatever res["trainingMetadata"] returns can be defined by that particular class. We don't care if the object is equal to None by its own definition, we want to know if the object is None.